core/src/main/scripts/importer/importOncokbDiscreteCNA.py

```diff
@@ -150,7 +156,11 @@ def get_cna_events(cna_file_path, server_url=None, alias_map=None):
         for sample_id in sample_ids:
             feature = {}
             feature[SAMPLE_ID] = sample_id
-            feature[ALTERATION] = int(line_elements[sample_indexes[sample_id]])
+            cna_value = line_elements[sample_indexes[sample_id]]
+            try:
+                feature[ALTERATION] = 0 if cna_value in cna_null_values else int(round(float(cna_value)))
+            except TypeError:
+                continue
```

Comment:
this is prob the most important change

Reply:
just FYI, the cbio importer also rounds non-integer values (e.g. 1.5 -> 2) so this follows that same logic.
Values other than -2, 1, 1, 2 are skipped by default.